TestAssets/TestProjects/docker-compose/docker-compose.dcproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.Docker.Sdk">
```

Comment:
@eerhardt Hi Eric, thanks for reviewing the change. This is a Docker-Compose project instead of a .NET Core project. For ToolsVersion not being necessary, is that a new feature in MSBuild or just for .NET Core project? Also does it apply to VS and VS Mac as well? We need the same project artifacts to be able to round trip between VS, VS Mac and CLI.